src/ui/wxWidgets/PasswordPolicy.cpp

```diff
@@ -24,7 +24,7 @@
 ////@end includes
 
 #include "PasswordPolicy.h"
-#include "PolicyManager.h"
+#include "../../core/PolicyManager.h"
```

Comment:
Should be "core/PolicyManager.h", to be consistent with the following line.

Reply:
I don't understand this one. Is it just for convention, or is the patch actually breaking anything?

I always thought an explicit path makes it better: compilation is faster since there's no searching for the header, you know while looking at the code where to find the header, can open the header easily from the editor (gf in vim), etc. etc.

Reply:
Convention and consistency.
With today's IDEs, there's no real need to specify the full relative path when -I directives have it. Also, this breaks when the including file's relative path changes (rare, but happens).